mod/developers/views/default/admin/developers/settings.php

```diff
@@ -5,82 +5,105 @@
 
 $data = [
```

Comment:
do these fields not belong in the form??

Reply:
not going to refactor the entire plugin, just adding a setting ;)